src/main/kotlin/uk/gov/gdx/datashare/services/AdminService.kt

```diff
@@ -0,0 +1,33 @@
+package uk.gov.gdx.datashare.services
+
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import uk.gov.gdx.datashare.enums.CognitoClientType
+import uk.gov.gdx.datashare.models.*
+
+@Service
+class AdminService(
+  private val cognitoService: CognitoService,
+  private val consumersService: ConsumersService,
+) {
+  fun createCognitoClient(cognitoClientRequest: CognitoClientRequest) =
+    cognitoService.createUserPoolClient(cognitoClientRequest)
+
+  @Transactional
+  fun createAcquirer(createAcquirerRequest: CreateAcquirerRequest) = cognitoService.createUserPoolClient(
+    CognitoClientRequest(createAcquirerRequest.clientName, listOf(CognitoClientType.ACQUIRER)),
+  )?.let {
+    val consumer = consumersService.addConsumer(ConsumerRequest(createAcquirerRequest.clientName))
```

Comment:
Do we have a uniqueness constraint on the client name? If so, would be worth moving above the cognito creation so we fail fast.